src/Button.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import styled from "styled-components";
+
+function Button(props) {
+  return (
+    <StyledButton type={props.type} value={props.value} {...props}>
+      {props.content}
```

Comment:
Instead of content use `props.children`

Whatever you wrap the button around will be its children.

`<Button>Submit</Button>` or `<Button><Icon /></Button>`... in these scenarios `'Submit'`or` <Icon />` would be the buttons children... it avoids adding an extra prop